### ISSUE TITLE
Updating domains does not save them

### ISSUE THREAD

**Issue by Rage65** [ctih1/frii.site-frontend#181]
**Describe the bug**
If you update the value of a subdomain then try to save it it says sucsuess but apon refresh of the dashboard it shows the old value. 

**To Reproduce**
update a subdomain
save
refresh

**Expected behavior**
When saving a domain it should commit to the database and stay on the new value even apon refresh of the dashboard

**Additional information:**
Device details 
Windows 10 
Firefox (x64)
DNS 1.1.1.1 and 8.8.8.8

**Comment by ctih1** [comment]
Does this happen with every domain, or just one?

**Comment by ctih1** [comment]
## A temporary fix
To fix this issue (atleast temporarily), re-register your domain

**Comment by Ponk446** [comment]
Had the same problem, the temp fix helps.

**Comment by ctih1** [comment]
Fixed in Dev.

Heres a rough sketch on what is going on

![image](https://github.com/user-attachments/assets/4c88ca27-538f-438a-a618-be0619af81bb)

**Comment by ctih1** [comment]
TL;DR: Uses wrong schema for database (caused by incorrect function call)

**Comment by ctih1** [comment]
task: implement auto fix for corrupted domains

**Comment by ctih1** [comment]
Fixed in Dev (canary.frii.site)
